iphone/Maps/Classes/Framework/MWMFrameworkListener.h

```diff
@@ -1,12 +1,15 @@
 #import "MWMFrameworkObservers.h"
 
+#include "platform/location.hpp"
+
 @interface MWMFrameworkListener : NSObject
 
 + (MWMFrameworkListener *)listener;
 + (void)addObserver:(id<MWMFrameworkObserver>)observer;
 
-@property (nonatomic, readonly) location::EMyPositionMode myPositionMode;
-@property (nonatomic) UserMark const * userMark;
+// TODO(AlexZ): We can avoid using this property at all by tuning and using LocationManager's
+// lastLocationIsValid method (reduce validity interval to 1 minute also will help to avoid other issues).
+@property (nonatomic) location::EMyPositionMode myPositionMode;
```

Comment:
Эн read/write сейчас, просто для хранения значения. Я написал в туду, что можно вообще избавиться от этого свойства.

> On Feb 24, 2016, at 10:39, Илья Гречухин <notifications@github.com> wrote:
> 
> In iphone/Maps/Classes/Framework/MWMFrameworkListener.h <https://github.com/mapsme/omim/pull/1957#discussion_r53903313>:
> 
> >  @interface MWMFrameworkListener : NSObject
> >  
> >  + (MWMFrameworkListener *)listener;
> >  + (void)addObserver:(id<MWMFrameworkObserver>)observer;
> >  
> > -@property (nonatomic, readonly) location::EMyPositionMode myPositionMode;
> > -@property (nonatomic) UserMark const * userMark;
> > +// TODO(AlexZ): We can avoid using this property at all by tuning and using LocationManager's
> > +// lastLocationIsValid method (reduce validity interval to 1 minute also will help to avoid other issues).
> > +@property (nonatomic) location::EMyPositionMode myPositionMode;
> Куда делся readonly?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/1957/files#r53903313>.
>